Documentation/ceph-dashboard-monitoring.md

```diff
@@ -0,0 +1,352 @@
+>```
+If you use the output URL in your browser, you should see the Prometheus monitoring website.
+
+![Prometheus Monitoring Website](media/prometheus-monitor.png)
+
+## Prometheus Ceph manager module
+
+The source of the Ceph cluster metrics is this manager module, by default is enabled, and the Prometheus operator service monitor configures Prometheus to scrape data from it.
+
+In order to avoid monitoring issues just verify that the scrape intervals are the same in Prometheus and in the Ceph manager module. Both intervals must be the same. Depending on your monitoring needs, a value between 5 and 30 is correct.
+
+In the Prometheus web UI, go to Status-Configuration. The scrape configuration should be the following:
+
+```yaml
+scrape_configs:
+- job_name: rook-ceph/rook-ceph-mgr/0
+  honor_timestamps: true
+  scrape_interval: 5s
+```
+
+To verify this value in the Ceph Manager module, open a [Rook toolbox](ceph-toolbox.md) and execute:
+```console
+ceph config get mgr mgr/prometheus/scrape_interval
+```
+>```
+>30.000000
+>```
+
+If you have a different value (as you can see in this example), you can change it executing:
+```console
+ceph config set mgr mgr/prometheus/scrape_interval 5
+```
+
+## Grafana
+
+The Grafana image used is a _customized_ image of Grafana V.6.7.4, and it includes the set of Grafana dashboards used in the Ceph dashboard, and a couple of visualization plugins.
+
+In order to deploy Grafana properly configured for Ceph, we will use the grafana.yaml file stored in the monitoring folder. Some modifications in this file could be needed: 
+
+**1. Change of SSL certificates (optional)**
+
+The grafana.yaml file includes a configmap for the SSL certificates used to enable secure access to the UI. These certificates are self-signed and if it is needed or preferred can be replaced by others.
+
+**2. Grafana storage (mandatory)**
+
+Grafana needs storage for its database and other stuff. The "grafana.yaml" file provided uses a PVC that get the storage from a PV based in a Storage Class with name "ceph-storage". _The user must be sure that a PV using this storage class is available for the grafana PVC_. Or change the storage class used in the grafana PVC to point the right one for your cluster. 
+
+Verify you have a PV with enough space(at least 500Mb) available for the Grafana PVC:
+```console
+kubectl get pv
+```
+>```
+>NAME              CAPACITY   ACCESS MODES   RECLAIM POLICY   STATUS      CLAIM   STORAGECLASS   REASON   AGE
+>ceph-pv-grafana   9Gi        RWO            Retain           Available           ceph-storage            5s
+>```
+
+_**Note about High Availability:**
+If you use a storage class that provides "high availability storage" (for example using a [Ceph shared filesystem](ceph-filesystem-md)), the Grafana installation can be also high available. Just is needed label more than 1 node with the label "app=grafana" and set the number of replicas to the desired number in the grafana.yaml file._
```

Comment:
Rook always recommends replica 3 for production pools. Seems like we don't need to mention this for high availability since Rook pools should already be configured as such.